src/components/NavBar/Navbar.css.ts

```diff
@@ -0,0 +1,34 @@
+import { style } from '@vanilla-extract/css'
+
+import { sprinkles } from '../../nft/css/sprinkles.css'
+
+export const nav = style([
+  sprinkles({
+    paddingX: '20',
+    paddingY: '12',
+    width: 'full',
+    height: '72',
+    zIndex: '2',
+    borderStyle: 'solid',
+  }),
+  {
+    borderColor: '#00000018',
```

Comment:
can we use some type of theme file for these? otherwise it won't work in light mode i think

Reply:
The lightmode for this color/component hasn't been designed yet

Reply:
in general can we avoid having hex colors like this though? if we can the `useTheme` or some other way to determine the color variables

Reply:
also chatted with @infredible, the light theme should be finalized also

Reply:
Our general practice has been if this is a one-off case (which is true for this border color) then you can hard code the hex, if it's used more than once you name and add it to the shared style file `src/nft/css/sprinkles.css.ts`

Reply:
Here are [the designs](https://www.figma.com/file/kNSDBMpOzxSTOP6MerohLm/Web-App-Spec-(Phase-0-%26-Phase-1)-%5BWIP%5D?node-id=4760%3A101171) for light/dark mode in the nav bar

In these updated designs, we shouldn't be using any hard-coded one-off colors